simulations/vip-244/simulations.ts

```diff
@@ -0,0 +1,249 @@
+import { expect } from "chai";
```

Comment:
We could add the generic check on the core pool, to confirm the main operations can de done after restoring the setter facet. Just in case